doc/releases/migration-guide-3.7.rst

```diff
@@ -45,6 +45,19 @@ Bluetooth HCI
 Controller Area Network (CAN)
 =============================
 
+* Removed the following deprecated CAN controller devicetree properties. Out-of-tree boards using
+  these properties need to switch to using the ``bus-speed``, ``sample-point``, ``bus-speed-data``,
+  and ``sample-point-data`` devicetree properties for specifying the initial CAN bitrate:
+
+  * ``sjw``
+  * ``prop-seg``
+  * ``phase-seg1``
+  * ``phase-seg1``
```

Comment:
@henrikbrixandersen fyi I will be fixing the two "seg1 instead of seg2" typos in a global migration guide / release notes cleanup PR later today

Reply:
Thanks